catalog/app/containers/Bucket/Search.js

```diff
@@ -339,15 +342,33 @@ export default function BucketSearch({
 }) {
   const isInStack = BucketConfig.useIsInStack()
   const { q: query = '', p, mode, ...params } = parseSearch(l.search)
+  const { preferences: prefs, result } = BucketPreferences.use()
   const page = p && parseInt(p, 10)
   const retry = (params.retry && parseInt(params.retry, 10)) || undefined
+  console.log(result, prefs)
```

Comment:
(:

Reply:
Hmm, with new setup husky+lint works for me. Don't know how it was sliped here